backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/AlertTypeService.java

```diff
@@ -27,9 +27,14 @@ public AlertTypeService(AlertTypeMapper mapper, AlertTypeRepository repository)
 	}
 
 	@Cacheable
-	public Optional<AlertType> findById(String id) {
+	public Optional<AlertType> readById(String id) {
 		Assert.hasText(id, "id is required; it must not be null or blank");
 		return repository.findById(id).map(mapper::fromEntity);
 	}
 
+	public AlertType readByCode(String code) {
```

Comment:
This should also return `Optional<AlertType>`.

Reply:
fixed